src/spec.c

```diff
@@ -1148,20 +1163,35 @@ void IndexSpec_FreeSync(IndexSpec *spec) {
 
 //---------------------------------------------------------------------------------------------
 
-void Indexes_Free(dict *d) {
-  arrayof(IndexSpec *) specs = array_new(IndexSpec *, 10);
+void Indexes_Free(dict *d, TrieMap *schemaPrefixes, void *aliases, void *cursors) {
+  if (!cleanPool) {
+    cleanPool = thpool_init(1);
```

Comment:
Why not initialise it on module load?

Reply:
We use the same pool with IndexSpec_Free for temporary indexes. Should we change on both?